### PR TITLE
(SIMP-8899) Remove vestiges of EL6

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-8899 #close

### CODE REVIEW

File: spec/classes/server_spec.rb [hunk 1]
```diff
@@ -2,88 +2,94 @@
...
-        end
+          context 'no_ppolicy' do
+            let(:params){{ :use_ppolicy => false }}
+
+            it { is_expected.to_not create_class('simp_openldap::slapo::ppolicy') }
+          end
 
-        context 'no_tcpwrappers' do
-          let(:params){{ :tcpwrappers => true }}
+          context 'no_tcpwrappers' do
```
Comment (op-ct) [inline spec/classes/server_spec.rb:88]:
```suggestion
          context 'tcpwrappers' do
```

**Review by op-ct — DISMISSED** [review]
👁️ 👍

File: spec/acceptance/suites/multi_tier/00_default_spec.rb [hunk 2]
```diff
@@ -215,8 +215,12 @@
 
   hosts.each do |server|
     context "on #{server}" do
+      install_simp_repos(server)
+
       let(:host) { server }
 
+      install_simp_repos(server)
+
```
Comment (lnemsick-simp) [inline spec/acceptance/suites/multi_tier/00_default_spec.rb:223]: duplicate?

Comment (trevor-vaughan) [reply spec/acceptance/suites/multi_tier/00_default_spec.rb:223]: No

**Review by lnemsick-simp — APPROVED** [review]

**Comment by lnemsick-simp** [comment]
Remaining artifacts:
manifests/server/conf.pp line 398 and the `slabd.erb` template that goes with it.

**Review by lnemsick-simp — APPROVED** [review]

**Pull request merged**